mobius/src/geometry/circle.rs

```diff
@@ -29,6 +29,16 @@ impl Circle {
     pub fn get_angle(&self, point: Complex) -> Option<f64> {
         (point - self.center).arg()
     }
+
+    pub fn point_inside(&self, point: Complex) -> bool {
+        (point - self.center).norm() <= self.radius * self.radius
+    }
+}
+
+impl PartialEq for Circle {
```

Comment:
Add some tests for this.

Reply:
I want to revisit constraints on all geometry primitives, I'll circle back to this later.